otelcollector/build/windows/scripts/main.ps1

```diff
@@ -106,46 +106,52 @@ function Set-EnvironmentVariablesAndConfigParser {
         [System.Environment]::SetEnvironmentVariable("AZMON_AGENT_CFG_FILE_VERSION", $config_file_version, "Machine")
     }
 
-    # Need to do this before the SA fetch for AI key for airgapped clouds so that it is not overwritten with defaults.
-    $appInsightsAuth = [System.Environment]::GetEnvironmentVariable("APPLICATIONINSIGHTS_AUTH", "process")
-    if (![string]::IsNullOrEmpty($appInsightsAuth)) {
-        [System.Environment]::SetEnvironmentVariable("APPLICATIONINSIGHTS_AUTH", $appInsightsAuth, "machine")
-    }
-    else {
-        Write-Host "Failed to set environment variable APPLICATIONINSIGHTS_AUTH for target 'machine' since it is either null or empty"
+    $customEnvironment = [System.Environment]::GetEnvironmentVariable("customEnvironment", "process").ToLower()
+    switch ($customEnvironment) {
+        "azurepubliccloud" {
+            $encodedaikey = "MWNkYTMxMTItYWY1Ni00ZmNiLWI4MDQtZjg5NDVhYTFjYjMy"
+            $aiendpoint = $null
+            Write-Host "setting telemetry output to the default azurepubliccloud instance"
+        }
+        "azureusgovernmentcloud" {
+            $encodedaikey = "ZmRjMTE0MmUtY2U0YS1mNTFmLWE4M2EtODBjM2ZjNDYwNGE5"
```

Comment:
Hi Kaveesh - i am not sure if scanning tools will pick up this ruby file and flag them. So i am thinking follow the old convention (set them as 4 different ENVs and pick the right one in this file) ?

Reply:
Sounds good Vishwa. I have updated the PR to reflect this and am building an image to test this. Once I verify that its working I will sync with you on this.

Reply:
Verified that this looks good.